Tasks/DeployVisualStudioTestAgent/make.json

```diff
@@ -2,13 +2,13 @@
      "externals": {
          "archivePackages": [
             {
-                "url": "https://testexecution.blob.core.windows.net/testplatformdeployment/3891278/TestPlatform.Deployment.zip",
+                "url": "https://testexecution.blob.core.windows.net/testplatformdeployment/4334139/TestPlatform.Deployment.zip",
```

Comment:
Is it just coincidence that both these packages have the same version number? Or is this a mistake?

Reply:
No co-incidence.. There are #signbuild1 which was made for test execution zip.. #signbuild2 which acq+testexecution zip.. So in git applied #singbuild two everywhere..